server/api/documents.js

```diff
@@ -7,16 +7,37 @@ router.get(
   "/",
   passport.authenticate("jwt", { session: false }),
   (req, res) => {
-    docsDb
-      .getAllDocuments()
-      .then(data => {
-        res.send(data);
-      })
-      .catch(err => {
-        console.error(err);
-        res.send(500);
-      });
+    let user = req.user.role;
+    console.log(user);
+
+    if (user) {
+      getIterpreterDocuments(res);
+    } else {
+      getUserDocuments();
```

Comment:
getUserDocuments expect to be passed a res variable isn't it?

Reply:
yes i did it